n/nekohtml/nekohtml_ubi8.5.sh

```diff
@@ -0,0 +1,56 @@
+# -----------------------------------------------------------------------------
+#
+# Package		: nekohtml
+# Version		: nekohtml-1.9.22
+# Source repo	: https://github.com/codelibs/nekohtml
+# Tested on	: ubi 8.5
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Sapna Shukla <Sapna.Shukla@ibm.com>
+# Language 		: Java
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#-----------------------------------------------------------------------------
+# PACKAGE_VERSION parameter for the script
+# nektml has tags as nekohtml-<version_number>
+# If the script is run without any parameter it builds and validate nekohtml-1.9.22
+# To run the script for other specific versions provide just the tag/version number ie., ./nekohtml_ubi8.4.sh 1.9.22
+#-----------------------------------------------------------------------------
+
+HOME_DIR=`pwd`
+PACKAGE_NAME=nekohtml
+PACKAGE_VERSION=${1:-1.9.22}
+PACKAGE_URL=https://github.com/codelibs/nekohtml
+
+mkdir -p output
+
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+yum install git maven -y
+
+#clone the repository
+git clone $PACKAGE_URL
+
+export WORK_DIR=$HOME_DIR/$PACKAGE_NAME
+cd $WORK_DIR
+CHECKOUT_VAL=nekohtml-$PACKAGE_VERSION
+git checkout $CHECKOUT_VAL
+
+if ! mvn clean install ; then
+	echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+	echo "$PACKAGE_NAME  |  $PACKAGE_VERSION  | $OS_NAME | GitHub | Fail |  Install_Fails"
+	exit 1
+else
+	echo "------------------$PACKAGE_NAME:install_success_no_test_availabe-------------------------"
```

Comment:
Do we need to update the echo message here ?
This package is having tests.

Reply:
On running **mvn test** , for nekohtml package, it returns no test to be run.
Therefore I have added the message there as no test available.

Highlighted that below 

[root@186fe13d1521 nekohtml]# mvn test
[INFO] Scanning for projects...
[INFO]
[INFO] -----------------< net.sourceforge.nekohtml:nekohtml >------------------
[INFO] Building Neko HTML 1.9.22
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ nekohtml ---
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] skip non existing resourceDirectory /home/nekohtml/src/main/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ nekohtml ---
[INFO] No sources to compile
[INFO]
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ nekohtml ---
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] skip non existing resourceDirectory /home/nekohtml/src/test/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ nekohtml ---
[INFO] No sources to compile
[INFO]
[INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ nekohtml ---
**[INFO] No tests to run.**
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.322 s
[INFO] Finished at: 2021-12-30T08:35:39Z
[INFO] ------------------------------------------------------------------------
[root@186fe13d1521 nekohtml]#